src/functionalTest/resources/json/checkAnswersMultipleExecutorsSummary.json

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
Perhaps we should add Applicant Alias details here too in our test
Also, must merge from master

Reply:
added new test for alias names